tests/test_forcing.py

```diff
@@ -0,0 +1,1025 @@
+from esmvalcore.experimental import get_recipe
```

Comment:
How do I run these tests? Pytest is not part of the package/environment I freshly installed

Reply:
Pytest is a dependency in `setup.py`, meaning it will be installed if you run `python setup.py develop`

Reply:
Hmm that's strange, I did pip install -e .[develop] and that didn't work.